frontend/components/common.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { shallow, render } from 'enzyme'
+import { expect } from 'chai'
+import * as common from './common'
+
+import './enzyme-conf.spec'
+
+describe('Common modules', function() {
+  describe('<GlobalStyle />', function() {
+    it('renders', function() {
+      shallow(<common.GlobalStyle />)
+    })
+  })
+
+  describe('<BaseFont />', function() {
+    it('renders', function() {
+      shallow(<common.BaseFont />)
+    })
+  })
+
+  describe('<H2 />', function() {
+    it('renders', function() {
+      shallow(<common.H2 />)
+    })
+  })
+
+  describe('<H3 />', function() {
+    it('renders', function() {
+      shallow(<common.H3 />)
```

Comment:
Might be just being slightly ignorant here, but what are these tests actually checking? What would I need to change/break to cause this test to fail?

Reply:
Not much. That the code doesn’t crash. Otherwise you have to play with the whole site to figure that out.